css/style.css

```diff
@@ -0,0 +1,94 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  font-family: sans-serif;
+  color: black;
+  margin: 100;
+  font-size: 1.5rem;
+  line-height: 1.2;
+  text-align: center;
+}
+
+.main-title {
+  font-size: 4.3rem;
+  margin-bottom: 5;
+}
+
+.subject {
+  margin: 3rem 0;
+  text-align: center;
+}
+
+h2 {
+  font-size: 3.5rem;
+}
+
+a {
+  color: #7daace;
+}
+
+ul {
+  max-width: 600px;
+  text-align: left;
+  line-height: 2rem;
+  margin: 0 auto 4rem auto;
+}
+
+li {
+  margin: 20px 10px;
+}
+
+a:visited {
+  color: #b9a4db;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: rgb(223, 223, 223);
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+  margin: 0 auto;
+  display: block;
```

Comment:
Kindly use flexbox because in modern CSS and taking the best practices into consideration we must use flexbox like in these situations where you have used a display block. 😄 😉